### PR TITLE
Added Known Browser Issues to FAQ

### PR DESCRIPTION
**Pull request by harshitOberoi**
**Issue #:**

**Description of changes:**
Added a new Section "Known Browser Issues" in FAQ

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
1) Open FAQ.html
2) check "Known Browser Issues" section

**Checklist:**

1. Have you successfully run `npm run build:release` locally?Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: guides/07_FAQs.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ The SDK is built to target ES2015, both syntax and features. If you need your bu
 
 Note that due to limitations in transpilers, requirements of the web platform might result in transpiled ES5 code that raises an error when run, such as "Please use the 'new' operator, this DOM object constructor cannot be called as a function". Prefer using ES2015 code on supported platforms.
 
+## Known Browser Issues
+Please refer https://github.com/aws/amazon-chime-sdk-js/issues/1059/ for all the known Browser Issues.
+
+### In Safari Audio stops playing as browser is minimized or move to background, is this a known issue?
+In [Safari](https://bugs.webkit.org/show_bug.cgi?id=231105) macOS Browser, Audio stops playing in background if audio source is binded to AudioContext API. To remediate this Issue please disable the WebAudio.
```
Comment (ltrung) [inline guides/07_FAQs.md:60]: Can we explicitly say the link to the bug? A bit random to make Safari as a link.

Comment (simmkyu) [inline guides/07_FAQs.md:60]: > macOS Safari has a [known issue](https://bugs.webkit.org/show_bug.cgi?id=231105) that a [MediaStreamAudioDestinationNode](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamAudioDestinationNode) stops working when you minimize the window. The bug also occurs when you activate the full-screen mode and switch to another window.
>
> [Amazon Voice Focus](https://aws.github.io/amazon-chime-sdk-js/modules/amazonvoice_focus.html) requires Web Audio. To remediate the issue, please disable Web Audio by passing the `{ enableWebAudio: false }` argument, or no argument, to `new DefaultDeviceController`.

We have to explain why disabling Web Audio helps and how to disable it.

**Review by simmkyu — CHANGES_REQUESTED** [review]

File: guides/07_FAQs.md (hunk 1 above)
Comment (jing-chen1) [reply guides/07_FAQs.md:60]: [nit]: this Issue -> this issue
[suggestion]: Could we add some details about the version, like "in Safari 14 or 15"?

Comment (harshitOberoi) [reply guides/07_FAQs.md:60]: Actually it is the same pattern used for all the bugs in the FAQ page.

File: guides/07_FAQs.md [hunk 2]
```diff
@@ -53,6 +53,12 @@ The SDK is built to target ES2015, both syntax and features. If you need your bu
 
 Note that due to limitations in transpilers, requirements of the web platform might result in transpiled ES5 code that raises an error when run, such as "Please use the 'new' operator, this DOM object constructor cannot be called as a function". Prefer using ES2015 code on supported platforms.
 
+## Known Browser Issues
```
[the remark this answers is not in the document]
Comment (harshitOberoi) [reply guides/07_FAQs.md:56]: moved the `Is the Amazon Chime SDK supported on mobile browsers?` to Browser Support.

[the remark this answers is not in the document]
Comment (harshitOberoi) [reply guides/07_FAQs.md:56]: Like Browser Support , Known Browser Issues should be a section in itself.

File: guides/07_FAQs.md (hunk 1 above)
Comment (harshitOberoi) [reply guides/07_FAQs.md]: It is the issue with both safari 14 and 15.

File: guides/07_FAQs.md [hunk 3]
```diff
@@ -53,6 +53,12 @@ The SDK is built to target ES2015, both syntax and features. If you need your bu
...
+Please refer https://github.com/aws/amazon-chime-sdk-js/issues/1059/ for all the known Browser Issues.
+
+### In Safari Audio stops playing as browser is minimized or move to background, is this a known issue?
```
[the remark this answers is not in the document]
Comment (harshitOberoi) [reply guides/07_FAQs.md]: done

File: guides/07_FAQs.md [hunk 4]
```diff
@@ -53,6 +53,12 @@ The SDK is built to target ES2015, both syntax and features. If you need your bu
 
 Note that due to limitations in transpilers, requirements of the web platform might result in transpiled ES5 code that raises an error when run, such as "Please use the 'new' operator, this DOM object constructor cannot be called as a function". Prefer using ES2015 code on supported platforms.
 
+## Known Browser Issues
+Please refer https://github.com/aws/amazon-chime-sdk-js/issues/1059/ for all the known Browser Issues.
```
[the remark this answers is not in the document]
Comment (harshitOberoi) [reply guides/07_FAQs.md]: done

File: guides/07_FAQs.md [hunk 5]
```diff
@@ -53,6 +53,23 @@ The SDK is built to target ES2015, both syntax and features. If you need your bu
...
+
+Amazon Chime SDK for JavaScript is supported on certain mobile browsers listed in the official Amazon Chime SDK documentation: https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-browsers. Developers can also build native mobile applications using the following SDKs (this option allows for meetings to continue when applications are sent to the background).
+
+Amazon Chime SDK for iOS https://github.com/aws/amazon-chime-sdk-ios
+
+Amazon Chime SDK for Android https://github.com/aws/amazon-chime-sdk-android
+
+
+## Known browser issues
+Please refer [Known Browser and Compatibility Issues](https://github.com/aws/amazon-chime-sdk-js/issues/1059/) for more information.
```
Comment (simmkyu) [inline guides/07_FAQs.md:67]: => `refer to `

File: guides/07_FAQs.md [hunk 6]
```diff
@@ -53,6 +53,23 @@ The SDK is built to target ES2015, both syntax and features. If you need your bu
...
+
+### In macOS Safari, audio stops playing when I minimize or move the window to the background. Is this a known issue?
+
+macOS Safari has a [known Web Audio issue](https://bugs.webkit.org/show_bug.cgi?id=231105) that a [MediaStreamAudioDestinationNode](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamAudioDestinationNode) stops working when you minimize the window. The bug also occurs when you activate the full-screen mode and switch to another window. To remediate the issue, disable Web Audio by passing the { enableWebAudio: false } argument, or no argument, to new DefaultDeviceController.
```
Comment (simmkyu) [inline guides/07_FAQs.md:71]: nit: Put the following two into \`code blocks\`.
- `{ enableWebAudio: false }`
- `new DefaultDeviceController`

**Review by simmkyu — APPROVED** [review]

**Pull request merged**